### PR TITLE
[Pre-clearance] Update institution profile

### LINKED ISSUE #682: [Pre-clearance] Update institution profile
- [x] Pre-clearance -- Update body text under "Financial institution details" @meissadia
- [x] Pre-clearance -- Add helper text under "Type of financial institution"
- [x] Pre-clearance -- Update body text under "Parent entity information (if applicable)"

### PR DESCRIPTION
**Pull request by meissadia**
Close #682 

|Section|Shot|
|---|---|
|View Financial institution details|Screenshot 2024-06-11 at 11 19 30 AM|
|Update Financial institution details|Screenshot 2024-06-11 at 11 19 39 AM|
|Type of FI helper text|update-types-helper-text|
|Parent entity intro|Screenshot 2024-06-10 at 9 43 56 AM|

### CODE REVIEW

**Comment by natalia-fitzgerald** [comment]
@meissadia 
I see an extra space in the last image after LOU

**Comment by meissadia** [comment]
> @meissadia I see an extra space in the last image after LOU. This was in the Figma file but I fixed.

Extra space removed. Screenshot updated.

File: src/pages/Filing/ViewInstitutionProfile/FinancialInstitutionDetails.tsx [hunk 1]
```diff
@@ -15,14 +14,31 @@ import { DisplayField } from './DisplayField';
 export const formatDomains = (domains?: Domain[]): string =>
   (domains ?? []).map((domain: Domain) => domain.domain).join(', ');
 
+const GetAnLEI = (
+  <Link href='https://www.gleif.org/en/about-lei/get-an-lei-find-lei-issuing-organizations'>
```
Comment (billhimmelsbach) [inline src/pages/Filing/ViewInstitutionProfile/FinancialInstitutionDetails.tsx:18]: I think this is going to be used elsewhere? Maybe move it over into the pile of other links?

File: src/pages/Filing/ViewInstitutionProfile/FinancialInstitutionDetails.tsx [hunk 2]
```diff
@@ -15,14 +14,31 @@ import { DisplayField } from './DisplayField';
...
+);
+
+export const updateInstitutionDescription = (
+  <>
+    To update the email domains for your financial institution,{' '}
+    <Link href='mailto:SBLHelp@cfpb.gov?subject=[BETA] Update financial institution profile: Update email domain'>
+      email our support staff
+    </Link>
+    . To update any other data in this section, contact your Local Operating
+    Unit (LOU) or visit {GetAnLEI} for a list of LOUs.
```
Comment (billhimmelsbach) [inline src/pages/Filing/ViewInstitutionProfile/FinancialInstitutionDetails.tsx:41]: I think this language hasn't been updated everywhere in figma yet, but I think this is supposed to be "visit GLIEF to identify your LOU"? Take a look at the figma under View your financial institution profile.

Screenshot 2024-06-11 at 10 14 49 AM


Screenshot 2024-06-11 at 10 15 21 AM

**Review by billhimmelsbach — DISMISSED** [review]
Heyo! Thanks for taking this, a small comment I'd like you to check on?

**Review by billhimmelsbach — APPROVED** [review]
Great!

**Pull request merged**